docs/home/contribute/participating.md

```diff
@@ -0,0 +1,75 @@
+
+## Roles and Responsibilities
+
+- Members
+
+Anyone who is a member of the Kubernetes organization can provide a review for a pull request. The SIG-DOCS team will frequently request a review for technical accuracy, and if you are a member, then you can indicate your approval by adding a comment of `/lgtm` to a pull request.
+
+- Reviewers
+
+Reviewers are individuals who have volunteered to provide their time to review documentation pull requests. You become a reviewer by adding your github handle to the [OWNERS file](https://github.com/kubernetes/community/blob/master/contributors/devel/owners.md) within the [website repo](https://github.com/kubernetes/website).
+
+Our automation will reviewers to use for reviews on pull requests, and a contributor may request your review with an `/assign [your_github_handle]` comment in the pull request. To indicate your approval of a pull request, a reviewer should add an `/approve` comment to the pull request. A reviewer may also use add a `/hold` comment on the pull request to prevent the pull request from being merged. Another reviewer or approver may also remove that hold with the comment `/hold cancel`.
+
+When a reviewer is assigned a pull request to review it is not a sole responsibility, and any other reviewer may also offer their opinions on the pull request. If a reviewer is requested, it is generally expected that the PR will be left to that reviewer to do their editorial pass on the content. For small and/or trivial pull requests, the assigned reviewer may be skipped if an approver thinks the content is sufficient and well understood. 
+
+- Approvers
+
+Approvers are the individuals with final approval over a PR for merging. An approver can indicate `/lgtm` or `/approve` to have a pull request merged, and all pull requests require at least one approver to provide their vote for the PR to be merged. In general, this is done with an `/approve` comment, expecting an `/lgtm` to come from another reviewer. 
+
+A special case is where a pull request is a quick fix, typo, etc - in those cases, an approver may enter `lgtm`, which is regarded as a combination of both and accepted without further review.
+
+### Teams and groups within SIG-DOCS
+
+You can get an overview of [SIG-DOCS from the community github repo](https://github.com/kubernetes/community/tree/master/sig-docs). The SIG-DOCS group has two teams defined on github:
+ - [@kubernetes/sig-docs-maintainers](https://github.com/orgs/kubernetes/teams/sig-docs-maintainers)
+ - [@kubernetes/sig-docs-pr-reviews](https://github.com/orgs/kubernetes/teams/sig-docs-pr-reviews)
+
+These groups are used for access to the [website repository](https://github.com/kubernetes/website), which houses the content that hosted at this site. Both can be referenced with their `@name` in github comments to communicate with everyone in that group.
```

Comment:
Replace with:
```
These groups maintain the [Kubernetes website repository](https://github.com/kubernetes/website), which houses the content hosted at this site. 
```
So:
> These groups maintain the [Kubernetes website repository](https://github.com/kubernetes/website), which houses the content hosted at this site. 

It's possible to use the aliases to contact the group as a whole, but it's almost never necessary, and I'd rather not encourage it.

Reply:
Funny enough: `sig-docs-*` are Kubernetes org teams, and have no actual permissions to kubernetes/website. I'll add them to the repo with Write access for `*-maintainers` and Read access for `*-reviews`.